include/seqan3/alignment/matrix/alignment_trace_matrix.hpp

```diff
@@ -98,29 +99,108 @@ struct alignment_coordinate
     size_t seq2_pos;
 };
 
-/*!\brief Compute the trace from a trace matrix
+/*!\brief Compute the number of insertions, deletions and
+ * substitutions between begin coordinate and end coordinate.
```

Comment:
I think substitutes is the wrong name since you don't know if bases are substituted (they could be the same and just match). Maybe call it `aligned` or `aligned_bases` or something else since substitutes implies mismatches.